config/locales/errors.en-GB.yml

```diff
@@ -41,3 +41,5 @@ en-GB:
       address:
         blank: "Address must be completed"
         too_long: "Address is too long"
+      privacy_notice:
+        blank: "Privacy notice must be checked"
```

Comment:
"checked" seems a bit specific, but wouldn't Rails give you a better message by default anyway?
 
> The default error message for this helper is "must be accepted". 

https://guides.rubyonrails.org/active_record_validations.html#acceptance

Reply:
yep, forgot to delete 🤦